tests/test_models.py

```diff
@@ -0,0 +1,87 @@
+import copy
+
+import pytest
+
+from tests.conf import example_task_list, random_task
+
+
+@pytest.fixture
+def setup_teardown():
```

Comment:
read about TestCase from Django and try integrate TestCase from Django with pytest